core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.html

```diff
@@ -133,6 +133,24 @@ <h4 class="oppia-translation-modal-author-header">
       </oppia-rte-output-display>
     </div>
   </section>
+  <section [hidden]="!reviewable"
+           class="oppia-image-review-panel"
+           *ngIf="explorationImagesString">
+    <div class="oppia-exploration-image-container float-left">
+      <oppia-rte-output-display *ngIf="!contentTypeIsSetOfStrings"
+                                  [rteString]="explorationImagesString"
+                                  [altTextIsDisplayed]="altTextIsDisplayed"
+                                  #contentPanel>
```

Comment:
Is it expected that the "oppia-rte-output-display" components displayed have the same template ref i.e. `#contentPanel` (there are 3 in this component with the same template ref ID)?

Reply:
Well, for this case I think it's fine to have the same template. Since we have included a *ngIf for this template, I think it's suffice as the diff is only a text section which shows alt text. WDYT?

Reply:
Ah, Sorry for the misunderstanding, I didn't get it in the first read. I have given it a different temp ref id to avoid ambiguity.